test/client.test.js

```diff
@@ -1899,7 +1899,7 @@ describe('ACC Client', function () {
     describe("Bearer token authentication", () => {
         // Bearer token authentication is used when embedding IMS for authentication
         it("Should create logged client", async() => {
-            const connectionParameters = sdk.ConnectionParameters.ofBearerToken("http://acc-sdk:8080", "$token$");
```

Comment:
Is this change related to this PR? I'm not sure I understand it: why calling ofBearerToken without a bearer token?

Reply:
This is only for code coverage